contract/data/mail_template.xml

```diff
@@ -63,6 +63,9 @@
         <p></p>
     </div>
     <p></p>
+    <p>Base url: ${object.get_base_url()}</p>
```

Comment:
Why putting these things visible in the template?

Reply:
Ups, sorry.